ckb-auth-rs/src/ckb_auth_dl.rs

```diff
@@ -146,3 +152,21 @@ pub fn ckb_auth_dl(
         _ => Err(CkbAuthError::RunDLError),
     }
 }
+
+pub fn ckb_auth_prepare(
+    entry: &CkbEntryType,
+    prefilled_data: &mut [u8],
```

Comment:
I think the high level API here lacks the functionality of determining how large a buffer is needed for `prefilled_data` as described in https://github.com/nervosnetwork/ckb-auth/blob/3937fed120fa0a4eb1c850e035924e6d93e1cf3a/docs/auth.md#L200-L203 . We may expose a function `get_required_prefilled_data_size(id &CkbAuthType)` that wrap around `int ckb_auth_load_prefilled_data(uint8_t auth_algorithm_id, uint8_t *prefilled_data, size_t *len)` with prefilled_data setting to NULL pointer and `*len` set to a pointer points to 0. If this function is not exposed, we need to at least describe the recommended prefilled_data size is `RECOMMEND_PREFILLED_LEN` in the doc.

Reply:
This should be re-designed. Send a PR about this changes.